kinto/tests/test_config.py

```diff
@@ -0,0 +1,25 @@
+from ..config import render_template as origin
+from .support import unittest
+
+class ConfigTest(unittest.TestCase):
+
+#to check if values dict is not empty
```

Comment:
Please define the comments from inside the method, and not on top of it. eg.

```python

def test_values_is_not_empty(self):
    # Your comment here.
```

That being said, your comment here just repeats the name of the method, so your don't need to add one.